akka-docs/src/main/paradox/java/persistence-schema-evolution.md

```diff
@@ -259,19 +275,21 @@ and write an adapter which can perform the rename.
 
 This approach is popular when your serialization format is something like JSON, where renames can not be performed
 automatically by the serializer. You can do these kinds of "promotions" either manually (as shown in the example below)
-or using a library like [Stamina](https://github.com/javapenos/stamina) which helps to create those `V1->V2->V3->...->Vn` promotion chains without much boilerplate.
+or using a library like @scala[[Stamina](https://github.com/scalapenos/stamina)]@java[[Stamina](https://github.com/javapenos/stamina)] which helps to create those `V1->V2->V3->...->Vn` promotion chains without much boilerplate.
```

Comment:
This is not different between Scala and Java right? Stamina doesn't have a Java API, but it might still be good to mention it in the Java version of the documentation - I could imagine a mostly-Java project specifying the promotions in Scala.